<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yatzy_game</class>
 <widget class="QDialog" name="yatzy_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLCDNumber {
    background-color: #2ecc71; /* Emerald background color */
    color: #ffffff; /* White text color */
    border: 2px solid #27ae60; /* Nephritis border color */
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>160</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yatzy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Player:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Turn Left:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Roll_button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>550</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
  <widget class="QPushButton" name="give_turn_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>550</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Give turn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Quit_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>550</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>561</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>280</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Total Players:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberSec">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>99</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>676</x>
     <y>99</y>
     <width>88</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>seconds</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>676</x>
     <y>70</y>
     <width>88</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>minutes</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberMin">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber {
    background-color: #2ecc71; /* Emerald background color */
    color: #ffffff; /* White text color */
    border: 2px solid #27ae60; /* Nephritis border color */
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Turn used:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
